src/documents/tests/test_file_handling.py

```diff
@@ -0,0 +1,441 @@
+import datetime
+import os
+import shutil
+from unittest import mock
+from uuid import uuid4
+from pathlib import Path
+from shutil import rmtree
+
+from dateutil import tz
+from django.test import TestCase, override_settings
+
+from django.utils.text import slugify
+from ..models import Tag, Document, Correspondent
+from django.conf import settings
+
+
+class TestDate(TestCase):
+    def tearDown(self):
```

Comment:
If we ever start running tests in parallel, which we currently don't seem to do, this will hurt us, as it will delete temporary directories of other tests, too. What do you think about making this forward-compatible and just remove `MEDIA_ROOT` by saving it on the instance? Something like `if hasattr(self, '_tmp_media_root'):` or something. Do you think that's overkill?

Reply:
Done